### PR TITLE
docker: Update README.md

### PR DESCRIPTION
**Pull request by hershpa**
Optimize and clean up the readme content for clarity.
Signed-off-by: Hersh Pathak <hersh.pathak@intel.com>

### CODE REVIEW

**Comment by hershpa** [comment]
@chaitanya1731, can you help review this PR?

File: docker/README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+`Note: ARG DTK_AUTO in the Dockerfile will be replaced automatically with the appropriate DTK image by KMM operator.`
 
-Intel GPU Drivers uses Dynamic Kernel Module Support (DKMS) to release the RPM packages and on-premises build the drivers modules on the RHEL server. With this Dockerfile, [Linux Kernel Build System](https://docs.kernel.org/kbuild/index.html) is used to build the driver modules.
+Intel GPU Drivers uses Dynamic Kernel Module Support (DKMS) to release the RPM packages and on-premise build the drivers modules on the RHEL server. With this Dockerfile, [Linux Kernel Build System](https://docs.kernel.org/kbuild/index.html) is used to build the driver modules.
 
-Multiple drivers are compiled and packaged into the single driver container image. To make sure the right drivers release combination are packaged, Intel GPU Driver version is used to select the version of each driver.
+Multiple drivers are compiled and packaged into the single driver container image. To ensure the correct driver release combination is packaged, Intel GPU Driver version is used to select the version of each driver.
```
Comment (uMartinXu) [inline docker/README.md:10]: Since we have several drivers packaged into the image. 
Should we use the correct "drivers" release?

Comment (hershpa) [reply docker/README.md:10]: The "driver release combination" implies that a combination consists of several driver releases.

Comment (chaitanya1731) [reply docker/README.md:10]: @uMartinXu "driver" is correct usage here.

Comment (uMartinXu) [reply docker/README.md:10]: Thanks! :=)

**Comment by uMartinXu** [comment]
I am ok with the PR. @chaitanya1731 What are your opinions on this PR? :=)

**Comment by chaitanya1731** [comment]
@uMartinXu PR Looks good to me.

**Comment by hershpa** [comment]
Based on the feedback gathered above, this PR is ready to merge.

**Pull request merged**